apps/service/geocoding/extract-city-from-geocoding-response.go

```diff
@@ -0,0 +1,75 @@
+package geocoding
+
+import "fmt"
+
+func extractCityFromGeocodingResponse(geocodingResp *GoogleGeocodingResponse, zipCode string, areaScope *AreaScope) (*CityResult, error) {
+
+	// Extract city from the first result
```

Comment:
remove, obsolete after refactor. feel free to add any function level comments above the function signature if you like.